Monika After Story/game/zz_calendar.rpy

```diff
@@ -879,13 +895,19 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
             cal_r_buttons = [
                 (
                     x.render(width, height, st, at),
-                    (x.xpos, x.ypos)
+                    (x.xpos, x.ypos),
+                    y.render(width, height, st, at),
+                    (x.xpos + self.DAY_BUTTON_WIDTH - y.render(width, height, st, at).get_size()[0] - 7, x.ypos + 5),
+                    z.render(width, height, st, at),
```

Comment:
since note text doesn't exist for every calendar date, this is a wasteful render (and arguably a wasteful generation of a Text object) when building these buttons. Rather than always building a Text object for note text, can you only create and render Text objects if we have actual text to show. 

Additionally, instead of creating a larger tuple for each day cell, I would restructure this loop to not use list comprehension and instead add the new renders to the list in the same format `(render, xy_tuple)`. That way the blit loop doesn't have to unpack so many items and doesn't need a branch for cases without note text.